### ISSUE TITLE
[SF][A11Y][ObjectStatus]: outline color won't change per preset states causing focus indicator not clearly visible and failing color contrast ratio

### ISSUE THREAD

**Issue by EvanWangJC** [UI5/webcomponents#12567]
### Bug Description

<ObjectStatus /> is given inverted and state with value in IndicationColor, for example:

`<ObjectStatus state={IndicationColor.Indication05} inverted interactive> Issue Happens When Focusing (Indication 05) </ObjectStatus>`

When focusing on it, outline color won't change per preset states in IndicationColor, causing focus indicator not clearly visible and failing color contrast ratio. This happens under IndicationColor.Indication05 and IndicationColor.Indication07

<img width="1509" height="710" alt="Image" src="https://github.com/user-attachments/assets/bae46f02-14c6-45cc-9515-e660f14fbf93" />

### Affected Component

ObjectStatus

### Expected Behaviour

Clear focus indicator for inverted ObjectStatus component.

### Isolated Example

https://stackblitz.com/edit/github-o8ow4gct-vpemayhu?file=src%2FApp.tsx

### Steps to Reproduce

1. Tab focusing on ObjectStatus (force focus state on <button />)
2. See unclear focus indicator for inverted ObjectStatus component

### Priority

Medium

### UI5 Web Components Version

2.14.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
